packages/sanes-chrome-extension/package.json

```diff
@@ -2,22 +2,26 @@
   "name": "sanes-chrome-extension",
   "version": "0.0.0",
   "private": true,
+  "config": {
+    "cs": "src/extension/contentscript.ts",
+    "bs": "src/extension/backgroundscript.ts"
+  },
   "scripts": {
     "use-config-test": "cp ./config/test.json ./public/assets/config/conf.json",
     "use-config-development": "cp ./config/development.json ./public/assets/config/conf.json",
     "use-config-production": "cp ./config/production.json ./public/assets/config/conf.json",
-    "build": "oza-react-scripts build",
+    "build": "react-scripts build $npm_package_config_cs $npm_package_config_bs",
```

Comment:
Looks like environment variables are not supported in Windows builds. `shx` does not support environment variables. Can we simplify this by inlining the file paths? Or do we need the ability to override the value?